internal/commands/ip_access/list.go

```diff
@@ -0,0 +1,82 @@
+package ipaccess
+
+import (
+	"fmt"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cli/user"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/spf13/pflag"
+)
+
+// CommandMetaList is the command meta for the `ip-access list` command
+var CommandMetaList = cli.CommandMeta{
+	Use:     "list",
+	Aliases: []string{"ls"},
+	Display: "allowed IPs list",
+	Hidden:  true,
+}
+
+// CommandList is the ip access list command
+type CommandList struct {
+	inputs listInputs
+}
+
+type listInputs struct {
+	cli.ProjectInputs
+}
+
+// Flags are the command flags
+func (cmd *CommandList) Flags(fs *pflag.FlagSet) {
+	cmd.inputs.Flags((fs))
+}
+
+// Inputs are the command inputs
+func (cmd *CommandList) Inputs() cli.InputResolver {
+	return &cmd.inputs
+}
+
+// Handler is the command handler
+func (cmd *CommandList) Handler(profile *user.Profile, ui terminal.UI, clients cli.Clients) error {
+	app, appErr := cli.ResolveApp(ui, clients.Realm, cmd.inputs.Filter())
```

Comment:
I will choose to not comment on the merits of using either `namedErr` or `err`, but I would say let's keep it consistent across the files worked on